src/scss/_dependencies.scss

```diff
@@ -50,13 +46,17 @@
  *
  * Object styles should be prefixed with `.o-`
  */
-@import 'objects/body';
-@import 'objects/buttons';
-@import 'objects/links';
-@import 'objects/lists';
-@import 'objects/tables';
-@import 'objects/form-controls';
-@import 'objects/form-toggle';
+@forward 'trumps/utilities';
```

Comment:
There was an error in a test project so had to move it up. For some reason now it isn't erroring, i'll take a look again and see if it can be moved back to where it originally was.

Reply:
Have moved it back. Seems to not error anymore weirdly.